### ISSUE TITLE
Error when importing using Gradle

### ISSUE THREAD

**Issue by nandorrb** [FolioReader/FolioReader-Android#97]
When I try to import using Gradle I get this:

/Users/mobile/.gradle/caches/transforms-1/files-1.1/folioreader-0.2.3.aar/79a625c2c7149e7fa5f1e9c550a5f7de/res/values/values.xml
Error:(111, 5) style attribute '@android:attr/windowEnterAnimation' not found
Error:(111, 5) style attribute '@android:attr/windowExitAnimation' not found
Error:(115, 5) style attribute '@android:attr/windowEnterAnimation' not found
Error:(115, 5) style attribute '@android:attr/windowExitAnimation' not found
Error:(119, 5) style attribute '@android:attr/windowEnterAnimation' not found
Error:(119, 5) style attribute '@android:attr/windowExitAnimation' not found
Error:(123, 5) style attribute '@android:attr/windowEnterAnimation' not found
Error:(123, 5) style attribute '@android:attr/windowExitAnimation' not found
Error:(128, 5) style attribute '@android:attr/windowEnterAnimation' not found
Error:(128, 5) style attribute '@android:attr/windowExitAnimation' not found
Error:(132, 5) style attribute '@android:attr/windowEnterAnimation' not found
Error:(132, 5) style attribute '@android:attr/windowExitAnimation' not found
Error:(136, 5) style attribute '@android:attr/windowEnterAnimation' not found
Error:(136, 5) style attribute '@android:attr/windowExitAnimation' not found
/Users/mobile/Documents/RepositorioAplicaciones/aplicacionesinstitutopacifico/Tests/ActualidadGubernamental/app/src/main/res/values/styles.xml
Error:(32, 5) style attribute '@android:attr/windowEnterAnimation' not found
Error:(32, 5) style attribute '@android:attr/windowExitAnimation' not found
Error:/Users/mobile/Documents/RepositorioAplicaciones/aplicacionesinstitutopacifico/Tests/ActualidadGubernamental/app/build/intermediates/incremental/mergePenalDebugResources/merged.dir/values/values.xml:1563 style attribute '@android:attr/windowEnterAnimation' not found
Error:/Users/mobile/Documents/RepositorioAplicaciones/aplicacionesinstitutopacifico/Tests/ActualidadGubernamental/app/build/intermediates/incremental/mergePenalDebugResources/merged.dir/values/values.xml:1567 style attribute '@android:attr/windowEnterAnimation' not found
Error:/Users/mobile/Documents/RepositorioAplicaciones/aplicacionesinstitutopacifico/Tests/ActualidadGubernamental/app/build/intermediates/incremental/mergePenalDebugResources/merged.dir/values/values.xml:1571 style attribute '@android:attr/windowEnterAnimation' not found
Error:/Users/mobile/Documents/RepositorioAplicaciones/aplicacionesinstitutopacifico/Tests/ActualidadGubernamental/app/build/intermediates/incremental/mergePenalDebugResources/merged.dir/values/values.xml:1575 style attribute '@android:attr/windowEnterAnimation' not found
Error:/Users/mobile/Documents/RepositorioAplicaciones/aplicacionesinstitutopacifico/Tests/ActualidadGubernamental/app/build/intermediates/incremental/mergePenalDebugResources/merged.dir/values/values.xml:1580 style attribute '@android:attr/windowEnterAnimation' not found
Error:/Users/mobile/Documents/RepositorioAplicaciones/aplicacionesinstitutopacifico/Tests/ActualidadGubernamental/app/build/intermediates/incremental/mergePenalDebugResources/merged.dir/values/values.xml:1584 style attribute '@android:attr/windowEnterAnimation' not found
Error:/Users/mobile/Documents/RepositorioAplicaciones/aplicacionesinstitutopacifico/Tests/ActualidadGubernamental/app/build/intermediates/incremental/mergePenalDebugResources/merged.dir/values/values.xml:1588 style attribute '@android:attr/windowEnterAnimation' not found
Error:/Users/mobile/Documents/RepositorioAplicaciones/aplicacionesinstitutopacifico/Tests/ActualidadGubernamental/app/build/intermediates/incremental/mergePenalDebugResources/merged.dir/values/values.xml:1592 style attribute '@android:attr/windowEnterAnimation' not found
Error:failed linking references
Error:java.util.concurrent.ExecutionException: com.android.builder.internal.aapt.AaptException: AAPT2 link failed:
Error:com.android.builder.internal.aapt.AaptException: AAPT2 link failed:
Error:Execution failed for task ':app:processPenalDebugResources'.
> Failed to execute aapt

**Comment by sethchhim** [comment]
I am unable to build too, I think there is a conflict with support library 26.0.0 because it built fine before.

@nandorrb Can you post your Android Studio version and Support Library version so I can confirm?

```
Error:java.lang.RuntimeException: com.android.builder.internal.aapt.AaptException: AAPT2 compile failed:
Error:Execution failed for task ':app:mergeDebugResources'.
> Error: java.lang.RuntimeException: com.android.builder.internal.aapt.AaptException: AAPT2 compile failed:
  aapt2 compile --legacy -o C:\Users\Seth Chhim\Documents\Viewer+forUbooquity\app\build\intermediates\res\merged\debug C:\Users\Seth Chhim\Documents\Viewer+forUbooquity\app\build\intermediates\incremental\mergeDebugResources\merged.dir\values\values.xml
  Issues:
   - ERROR: C:\Users\Seth Chhim\Documents\Viewer+forUbooquity\app\build\intermediates\incremental\mergeDebugResources\merged.dir\values\values.xml:832 duplicate value for resource attr/font with config '
   - ERROR: C:\Users\Seth Chhim\Documents\Viewer+forUbooquity\app\build\intermediates\incremental\mergeDebugResources\merged.dir\values\values.xml:760 resource previously defined here
```

**Comment by mahavir155** [comment]
@sethchhim can you check if new version 0.3.1 works for you?

**Comment by sethchhim** [comment]
@mahavir155 It buids fine in Android Studio 2.3.3 but if I use Android 3 Beta 2, it will fail to build due to error of `duplicate value for resource attr/font with config` and `resource previously defined here`. I think it might work if you update latest support library version to `26.0.1`. I will continue to check.

**Comment by BrianLusina** [comment]
I am encountering the same issue, even with the support library of 26.0.1. Is there a way to make this work with this support library version? I can't seem to find a workaround

**Comment by jorgevalbuena56** [comment]
I am having the same issue any solution?

**Comment by saurabhthorat** [comment]
Has anyone found a fix for this problem?

**Comment by BrianLusina** [comment]
I forked the library and had to comment [this](https://github.com/FolioReader/FolioReader-Android/blob/b2a14a781ef3011bb5be6a0abdcf3a315b94d255/folioreader/res/values/attrs.xml#L3) out for it to work.

**Comment by morteza-moradi** [comment]
I am having the same issue any solution?

**Comment by arvindvyas** [comment]
this fixed when I upgrade my gradel to 4.1
